xbmc/network/WebServer.cpp

```diff
@@ -765,6 +766,10 @@ int CWebServer::CreateFileDownloadResponse(const std::shared_ptr<IHTTPRequestHan
   std::shared_ptr<XFILE::CFile> file = std::make_shared<XFILE::CFile>();
   std::string filePath = handler->GetResponseFile();
 
+  // white/black list access check
+  if (!CFileUtils::ZebraListAccessCheck(filePath))
+    return SendErrorResponse(request.connection, MHD_HTTP_NOT_FOUND, request.method);
```

Comment:
MHD_HTTP_FORBIDDEN implies a hit which gives a hint that it is there,
MHD_HTTP_NOT_FOUND gives no info about the presence or absence.
I figured, this is a security issues, why provide additional info. Could be either, dealers choice.